README.md

```diff
@@ -6,26 +6,40 @@ This readme explains the reasoning behind the choice to use the hybrid (rather t
 ## Run it
 
 ### Setup
-
+Clone repository
 ```bash
-pipenv install
-pipenv shell
+$ git clone git@github.com:BONSAMURAIS/EXIOBASE-conversion-software.git
+$ cd EXIOBASE-conversion-software
 ```
 
-### Convert `xslb` to `csv`
+Create pipenv
+
+```bash
+$ pipenv install
+$ pipenv shell
 ```
-python scripts/excel2csv.py -i data/MR_HSUP_2011_v3_3_17.xlsb -o data/
 
-python scripts/excel2csv.py -i data/MR_HUSE_2011_v3_3_17.xlsb -o data/
+### Install
+```bash
+$ python setup.py install
 ```
 
-### Convert `csv` to `ttl`
+
+## Usage
+### As a command line tool
+### Convert `xslb` to `csv`
 ```
-python scripts/csv2rdf.py -i data/MR_HSUP_2011_v3_3_17.csv -o data/  -c HSUP --flowtype output
-python scripts/csv2rdf.py -i data/MR_HUSE_2011_v3_3_17.csv -o data/  -c HUSE --flowtype input
+$ excel2csv-cli -i data/MR_HSUP_2011_v3_3_17.xlsb -o data/
```

Comment:
could this be just `excel2csv` without `-cli` ? Same for the other command?

Reply:
@kuzeko  We sure could, the `-cli` extension is however currently used for the Arborist script, so following that standard, cli tools should end with `-cli`.

Reply:
I see, ok then